src/App.tsx

```diff
@@ -21,11 +38,19 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={itemsPerPageValue}
+            onChange={(event) => {
+              setItemsPerPage(+event.target.value);
+            }}
```

Comment:
it is bad practise to make anonymous arrow functions in jsx. better to make handler and pass reference her
please pay attention on it in next tasks
```
const onChangeSelect = (event: React.ChangeEvent<HtmlSelectElement>) => {
  setItems(+event.target.value);
}
```
```suggestion
            onChange={onChangeSelect}
```